src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import './TodoList.scss';
+
+import { Todo } from '../../types';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
```

Comment:
Consider using a more descriptive class name for the section element, like `todo-list-container`. 📝